### PR TITLE
Convert ReportingObserver article and update

### PR DESCRIPTION
**Pull request by jpmedley**
This does two things:

* Converts an existing article on ReportingObserver from WebFu to web.dev.
* Adds a mention that workers now [support it in Chrome 84](https://www.chromestatus.com/features/6310316751192064).

This needs to be lie on web.dev by the end of business on **Wednesday, May 27**. This is a hard deadline.

### CODE REVIEW

**Comment by jpmedley** [comment]
@robdodson I'm having a little trouble with `run npm test`. Please advise.

File: src/site/_data/contributors.js [hunk 1]
```diff
@@ -1082,6 +1082,16 @@ const contributors = {
       name: 'Google',
     },
   },
+  ericbidelman: {
+    name: {
+      given: 'Eric',
+      family: 'Bidelman',
+    },
+    country: 'US',
```
Comment (robdodson) [inline src/site/_data/contributors.js:1090]:
```suggestion
    twitter: 'ebidel',
    country: 'US',
```

Comment (robdodson) [reply src/site/_data/contributors.js:1090]: LGTM with a suggestion to add Eric's twitter profile.

File: src/site/content/en/blog/reporting-observer/index.md [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+}, {buffered: true});
+
+observer.observe();
+```
+
+Use the callback to send reports to a backend or analytics provider for
+analysis.
+
+Why is that useful? Until this API, deprecation and intervention warnings were
+only available in the DevTools as console messages. Interventions, in
```
Comment (philipwalton) [inline src/site/content/en/blog/reporting-observer/index.md:37]: s/the DevTools/DevTools/ (right?)

File: src/site/content/en/blog/reporting-observer/index.md [hunk 3]
```diff
@@ -0,0 +1,216 @@
...
+particular, are only triggered by various real-world constraints like device and
+network conditions. Thus, you may never even see these messages when
+developing/testing a site locally. `ReportingObserver` provides the solution to
```
Comment (philipwalton) [inline src/site/content/en/blog/reporting-observer/index.md:40]: Maybe change "the solution" to "a solution"?

File: src/site/content/en/blog/reporting-observer/index.md [hunk 4]
```diff
@@ -0,0 +1,216 @@
...
+<figure>
+  <img src="./consolewarnings.png"
+       class="screenshot" alt="DevTools console warnings for deprecations and interventions."
+       title="DevTools console warnings for deprecations and interventions.">
+  <figcaption>Browser-initiated warnings in the DevTools console.</figcaption>
+</figure>
+
+One would naturally think `window.onerror` captures these warnings. It does not.
+That's because `window.onerror` does not fire for warnings generated directly by
+the user agent itself. It fires for runtime errors (JS exceptions and syntax
```
Comment (philipwalton) [inline src/site/content/en/blog/reporting-observer/index.md:79]: s/JS/JavaScript/

File: src/site/content/en/blog/reporting-observer/index.md [hunk 5]
```diff
@@ -0,0 +1,216 @@
...
+});
+
+observer.observe();
+```
+
+## Conclusion
+
+`ReportingObserver` gives you an additional way for discovering and monitoring
+potential issues in your web app. It's even a useful tool for understanding the
+health of your code base (or lack thereof). Send reports to a backend, know
```
Comment (philipwalton) [inline src/site/content/en/blog/reporting-observer/index.md:185]: s/backend/back end/

Comment (kaycebasques) [reply src/site/content/en/blog/reporting-observer/index.md:185]: @philipwalton style guide convention is to write it as one word: https://developers.google.com/style/word-list#backend

File: src/site/content/en/blog/reporting-observer/index.md (hunk 2 above)
Comment (kaycebasques) [reply src/site/content/en/blog/reporting-observer/index.md:37]:
```suggestion
only available in DevTools as console messages. Interventions, in
```
@philipwalton y u no use the [Suggestion UI](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request#adding-line-comments-to-a-pull-request) (step 6)

File: src/site/content/en/blog/reporting-observer/index.md (hunk 3 above)
Comment (kaycebasques) [reply src/site/content/en/blog/reporting-observer/index.md:40]:
```suggestion
developing/testing a site locally. `ReportingObserver` provides a solution to
```

File: src/site/content/en/blog/reporting-observer/index.md (hunk 4 above)
Comment (kaycebasques) [reply src/site/content/en/blog/reporting-observer/index.md:79]:
```suggestion
the user agent itself. It fires for runtime errors (JavaScript exceptions and syntax
```

File: src/site/content/en/blog/reporting-observer/index.md (hunk 2 above)
Comment (jpmedley) [reply src/site/content/en/blog/reporting-observer/index.md:37]: Fixed.

File: src/site/content/en/blog/reporting-observer/index.md (hunk 3 above)
Comment (jpmedley) [reply src/site/content/en/blog/reporting-observer/index.md:40]: Fixed.

File: src/site/content/en/blog/reporting-observer/index.md (hunk 4 above)
Comment (jpmedley) [reply src/site/content/en/blog/reporting-observer/index.md:79]: Fixed.

File: src/site/content/en/blog/reporting-observer/index.md (hunk 5 above)
Comment (jpmedley) [reply src/site/content/en/blog/reporting-observer/index.md:185]: Fixed.

**Comment by jpmedley** [comment]
@philipwalton Feedback addressed. PTAL.

**Comment by philipwalton** [comment]
LGTM

**Comment by jpmedley** [comment]
@kaycebasques PTAL

**Pull request merged**